### PR TITLE
Low stock quantity

### PR DESCRIPTION
**Pull request by ArturSprinter**
Added low stock quantity field in model, which now will allow users to configure when client will see 'low stock' availability status.

Previous 'in stock' and 'out of stock' nop statuses are nice, but most store engines provides also 'low stock' state to inform clients they should consider buying product now (because product may disappear in near future from store).

### CODE REVIEW

File: src/Libraries/Nop.Core/Domain/Catalog/ProductStockStatus.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
```
Comment (DmitriyKulagin) [inline src/Libraries/Nop.Core/Domain/Catalog/ProductStockStatus.cs]: Remove unnecessary Usings

File: src/Libraries/Nop.Core/Domain/Catalog/ProductStockStatus.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace Nop.Core.Domain.Catalog
+{
+    public enum ProductStockStatus
```
Comment (DmitriyKulagin) [inline src/Libraries/Nop.Core/Domain/Catalog/ProductStockStatus.cs]: Add summary

**Comment by DmitriyKulagin** [comment]
@ArturSprinter This is a good idea, but we have to take care of how best to implement it. And since the work here is not complete, I created a new issue (#5089) for this. You can continue your development with a new pull request. Thanks.

**Comment by ArturSprinter** [comment]
Thanks.
I will try to continue work on this feature, if I got some spare time.

**Pull request closed without merging**